projects/prometheus/prometheus/README.md

```diff
@@ -1,18 +1,35 @@
-![Version](https://img.shields.io/badge/version-v2.53.1-blue)
+![Version](https://img.shields.io/badge/version-v2.54.0-blue)
```

Comment:
Why are we updating this version here in the README? It is still the old version v2.53.1 in the [GIT_TAG](https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/prometheus/prometheus/GIT_TAG) file.

Reply:
This was supposed to get merged first
https://github.com/aws/eks-anywhere-build-tooling/pull/3671